### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by Muddz** [quarkusio/quarkus#36879] [Q&A]
-

[a comment by an automated account is not shown]

**Comment by brunobat** [comment]
@Muddz I'm unaware of those properties. What do you want to get? Get the user id on the spans?
If yes, you can use this property:
[`quarkus.otel.traces.eusp.enabled=true`](https://quarkus.io/version/main/guides/opentelemetry#quarkus-opentelemetry_quarkus.otel.traces.eusp.enabled)
It will add user and role data to the span.

**Reply by Muddz** [reply]
I found out that these properties came from a inhouse custom library that used naming that made it look like it came from quarkus
